scripts/parts.py

```diff
@@ -51,11 +43,8 @@
         elif args.levels_testing == 'levels.tst':
             args.levels_testing = os.path.join(mc.test_data_path, 'levels.tst')
 
-    state_args = {'verbosity':args.verbosity.lower(),
-                  'mapr_cols':args.mapr_cols.lower(),
-                  'show_levels':args.show_levels,
-                  'show_state':args.show_state,
-                  'levels_testing':args.levels_testing}
+    state_args = {'verbosity':args.verbosity,
+                  'show_state':args.show_state}
```

Comment:
The `state_args` variable does not appear to be used at all in this file anymore, so you can just delete it altogether.